README.md

```diff
@@ -50,7 +50,7 @@ this repository may be used on the following networks:
 
 - Clone the repo
 - Install prerequisites:
-  - [node.js](https://nodejs.org/) v16.10.0+
+  - [node.js](https://nodejs.org/) v16+
```

Comment:
When I see v16+, it makes me think any version >= 16 is okay (i.e. v18 would be okay too).
Is this what we actually mean?
```suggestion
  - [node.js](https://nodejs.org/) Any v16 version
```

Reply:
Yeah, this time I messed up a bit. That was my intension to precise supported node version. I totally agree with your suggestion